### PR TITLE
Categories branch

### PR DESCRIPTION
**Pull request by ntazelo**
### Added features

**Fetch category images and display them on the home page**
**Fetch category details then display them to the details**
**Add test for the category reducer**
[link to loom presentation](https://www.loom.com/share/9f907c93cc2a416282267d563dc51f23)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+- Javascript
+- Html
+
+# Getting Started
+
+If you're new to GitHub and just want to download the capinterest app, press the green button saying "Code", and then choose the "Download ZIP" option or clone it locally using this command : **git clone https://github.com/ntazelo/capinterest.git** 
+
+- After cloning install the packages by running `npm install` and for starting it run `npm start`.
+
+
```
Comment (ArthurGC) [inline README.md:23]: - [ ] You want to consider adding a live preview version for the project(deployment) and paste its link in your README file too. 😉

File: src/Components/Categories/Categories.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
+.categories-cont {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(9, 60vw);
+}
+
```
Comment (ArthurGC) [inline src/Components/Categories/Categories.css:6]: - [ ] As the requirements show, you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). 

image

For colors, the design only has 2 of those.
image

For typography, there are 2 font families
image

File: src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+              <App />
+            </Provider>
+          </BrowserRouter>
+        </React.StrictMode>, container,
+      );
+    });
+
+    expect(pretty(container.innerHTML)).toMatchSnapshot();
+  });
+});
```
Comment (ArthurGC) [inline src/App.test.js:38]: - [ ] Following the requirements:

image
It is necessary to create integration tests for your project. Those tests have a relationship with components that use useEffect to fetch data when they render and send action to the reducer to update your global state. 😉

**Review by ArthurGC — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
Hi  @ntazelo 

Amazing job!
There are some issues that you still need to work on to go to the next project but you are almost there! 



## Good Points :+1: :
You used proper GitHub Flow :heavy_check_mark:
Your linter checks are all passing :heavy_check_mark:
Your PR messages are descriptive :heavy_check_mark:
Your commit messages are meaningful :heavy_check_mark:

## Aspects to improve ♻️
_Check the comments under the review._
- [ ] Your video is not showing a piece of the code that you find interesting or are proud of making. It would be better to create a short video to complete this requirement. ⚒️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by er-contreras — APPROVED** [review]
Hi @ntazelo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did most of the work, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## 🔍
- Consider improving your video presentation highlighting part of your code that was interesting for you.
- Add more tests to your project.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ntazelo** [comment]
Thank you @eri8-9 for the review I will do your suggestion.

**Pull request merged**